interface_config.js

```diff
@@ -38,6 +38,11 @@ var interfaceConfig = {
      */
     AUTHENTICATION_ENABLE: true,
 
+    /**
+     * If you are alone in the meeting, show an invite prompt in the header
+     */
+    SHOW_INVITE_MORE_HEADER: true,
```

Comment:
Can you please reverse this and call it `HIDE_INVITE_MORE_HEADER` defaulting to `false`. Otherwise those updating but not changing interface_config.js will get different behavior.

Reply:
I'll make the updates